.travis.yml

```diff
@@ -36,9 +39,9 @@ matrix:
           # Test LTS versions. This makes sure we do not use Symfony packages with version greater
           # than 2 or 3 respectively.
         - php: 7.2
-          env: DEPENDENCIES="symfony/lts:^2 symfony/force-lowest:~2.8.0 zendframework/zend-diactoros:^1.4.1"
+          env: DEPENDENCIES="$DEPENDENCIES symfony/lts:^2 symfony/force-lowest:~2.8.0"
         - php: 7.2
-          env: DEPENDENCIES="symfony/lts:^3 symfony/force-lowest:~3.4.0 zendframework/zend-diactoros:^1.4.1"
+          env: DEPENDENCIES="$DEPENDENCIES symfony/lts:^3 symfony/force-lowest:~3.4.0"
```

Comment:
Are these two changes needed?

Reply:
But http-interop/http-factory-diactoros is already in the global. Right? 
So no need to add it.

Reply:
But if we overwrite the dependencies here without including the previous value, this configuration will mark `DiactorosFactoryTest` and `PsrHttpFactoryTest` as skipped because `zendframework/zend-diactoros` and `http-interop/http-factory-diactoros` won't be installed.

Reply:
Hehe. I’ve been reviewing this PR multiple times but I’ve always read this change wrong. 

Thank you for you explanation and your patience. Of course, this change is valid.